spec/unit/resources/matchers_spec.rb

```diff
@@ -41,6 +41,20 @@
     expect(chef_run).to_not reset_oneview_ethernet_network_connection_template('')
     expect(chef_run).to_not delete_oneview_ethernet_network('')
 
+    # oneview_firmware
+    expect(chef_run).to_not add_oneview_firmware('')
+    expect(chef_run).to_not remove_oneview_firmware('')
+    expect(chef_run).to_not create_oneview_firmware_custom_spp('')
+
+    # oneview_connection_template
+    expect(chef_run).to_not update_oneview_connection_template('')
+    expect(chef_run).to_not reset_oneview_connection_template('')
+
+    # oneview_network_set
+    expect(chef_run).to_not create_oneview_network_set('')
+    expect(chef_run).to_not create_oneview_network_set_if_missing('')
+    expect(chef_run).to_not delete_oneview_network_set('')
```

Comment:
Not sure if you meant to add this in here or not, but it's actually defined below also (in alphabetical order)